db/db_compaction_test.cc

```diff
@@ -767,10 +793,13 @@ TEST_F(DBCompactionTest, DisableStatsUpdateReopen) {
     if (options.skip_stats_update_on_db_open) {
       // If update stats on DB::Open is disable, we don't expect
       // deletion entries taking effect.
-      ASSERT_LT(db_size[0] / 3, db_size[2]);
+      ASSERT_LE(db_size[0], db_size[2]);
```

Comment:
This one I'm most worried about. Open still does a flush that can trigger compaction, irrespective of deletion entries.

Reply:
Fixed.